src/Arcus.Security.Tests.Integration/IntegrationTest.cs

```diff
@@ -27,7 +27,7 @@ public IntegrationTest(ITestOutputHelper testOutput)
             var configuration = new LoggerConfiguration()
                 .WriteTo.XunitTestLogging(testOutput)
                 .WriteTo.Sink(InMemoryLogSink)
-                .WriteTo.AzureApplicationInsights(Configuration.GetValue<string>("Arcus:ApplicationInsights:InstrumentationKey"));
+                .WriteTo.AzureApplicationInsightsWithInstrumentationKey(Configuration.GetValue<string>("Arcus:ApplicationInsights:InstrumentationKey"));
```

Comment:
I think it would be good if we could use the ConnectionString approach here instead of Instrumentation-Key.
Something for another PR ?

Reply:
Yes, could be. 👍 Just upgraded the one with the same functionality here.
Created an issue for this: #428